.github/workflows/ci.yml

```diff
@@ -107,7 +107,7 @@ jobs:
     runs-on: ubuntu-18.04
     strategy:
       matrix:
-        tool: [Haplotypecaller, Freebayes, Manta, mpileup, Strelka, TIDDIT]
+        tool: [Haplotypecaller, Freebayes, Manta, mpileup, Strelka, TIDDIT, Ascat]
```

Comment:
Would that be it?

Reply:
Unfortunately no, we would actually need some test data that countains copy number variation, and generate ascat loci and gc files for that data as well.
I'm afraid it'll be slightly more complicated.
We can work that out in another PR I think it'll be safer

Reply:
ahh, so sorry for this messy PR